modules/tools/vault-less-secure/values.tpl

```diff
@@ -33,6 +31,22 @@ server:
     limits:
       memory: 256Mi
       cpu: 512m
+  serviceAccount:
+    create: false
+    name: vault
+
+  extraContainers:
+    - name: vault-init
+      image: harbor.parker.gg/library/vault-init:v1
```

Comment:
todo: replace with actual docker image once https://github.com/liatrio/vault-init/pull/1 is merged and released